build.gradle.kts

```diff
@@ -135,15 +148,16 @@ publishing {
                 url.set("http://github.com/MicroUtils/kotlin-logging/tree/master")
             }
         }
-        artifact(tasks["javadocJar"])
+        // TODO: Fix generation of Java Doc Jars.
```

Comment:
This should be fixed before merge

Reply:
It is important to note that there are many errors that appear after running the **dokka** Gradle task.

Reply:
Below are some of the elements that dokka cannot find:

- Class:KMarkerFactory in
- Class:KotlinLogging in
- Interface:KLogger in
- Function:getMarker in
- Field:INSTANCE in
- Function:logger in
- Function:trace in
- Function:debug in
- Function:info in
- Function:warn in
- Function:error in
- Function:entry in
- Function:exit in
- Function:throwing in
- Function:catching in

Reply:
Looks as though there are many parts of the public API that haven't been documented to Dokka's satisfaction.

Reply:
Can apply the fix in the build file however the build itself will fail CI tests.

Reply:
ok, you're welcome to open another issue for the dokka errors. the main reason we need (other than documentation itself) it is that maven central don't allow to publish without docs jars).